src/app.js

```diff
@@ -17,6 +18,35 @@ app.use(encryptCookieNodeMiddleware(process.env.SECRET_KEY));
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
+const options = {
```

Comment:
if these options are static lets no have it in our app.js and move it to constant file, and the naming can be SWAGGER_OPTIONS